packages/nx-firebase/package.json

```diff
@@ -22,10 +22,9 @@
   "executors": "./executors.json",
   "peerDependencies": {
     "@nx/devkit": ">= 16.1.1",
+    "@nx/node": "^16.6.0",
```

Comment:
I'm thinking this should be ">= 16.1.1" ?

Or even more ideally, remove it altogether. The plugin init generator already tries to ensure `@nx/node@workspace-nx-version` is a workspace dependency when it runs.